modules/apps/27-interchain-accounts/host/ibc_module_test.go

```diff
@@ -148,10 +149,10 @@ func (suite *InterchainAccountsTestSuite) TestOnChanOpenTry() {
 		},
 		{
 			"account address generation is block dependent", func() {
-				icaHostAccount := icatypes.GenerateAddress(suite.chainB.GetContext(), path.EndpointB.ConnectionID, path.EndpointA.ChannelConfig.PortID)
-				err := suite.chainB.GetSimApp().BankKeeper.SendCoins(suite.chainB.GetContext(), suite.chainB.SenderAccount.GetAddress(), icaHostAccount, sdk.Coins{sdk.NewCoin("stake", sdkmath.NewInt(1))})
```

Comment:
We longer use `SendCoins` to front run account creation in tests. This functionality in sdk is removed. The x/bank module will holds funds in the address space but the account will not be created. Instead we can create the account directly with x/auth in test code